recipe/meta.yaml

```diff
@@ -1,5 +1,4 @@
 {% set version = "0.9.1" %}
-{% set number = 1 %}
```

Comment:
is this version of torchvision compatible with pytorch 1.9?

Reply:
Upstream doesn't constrain the version [directly](https://github.com/pytorch/vision/blob/v0.9.1/setup.py#L61), and this feedstock had no upper pytorch requirement for as long that I'm aware. If the CI passes, I'd say it's fine.